quodlibet/ext/events/synchronize_to_device.py

```diff
@@ -741,7 +742,8 @@ def _run_preview(self):
         for root, __, files in os.walk(self.expanded_destination):
             for name in files:
                 file_path = os.path.join(root, name)
-                if file_path not in export_paths:
+                if file_path not in export_paths and \
+                        not file_path.__contains__("cover.jpg"):
```

Comment:
```suggestion
                        "cover.jpg" not in file_path:
```

Reply:
Oh yes, the elegance of Python! I do not use it much anymore, thanks for your feedback!